File: android/src/main/java/com/getcapacitor/community/classes/events/FoundEvent.java
package com.getcapacitor.community.classes.events;

import androidx.annotation.NonNull;
import com.getcapacitor.JSObject;

public class FoundEvent extends PeerIDEvent {

    @NonNull
    private final String message;

    public FoundEvent(@NonNull String peerID, @NonNull String message) {
        super(peerID);
        this.message = message;
    }

    @NonNull
    public JSObject toJSObject() {
        JSObject result = super.toJSObject();

        result.put("message", message);

        return result;
    }
}

File: ios/Sources/BitchatPlugin/Classes/Events/FoundEvent.swift
import Foundation
import Capacitor

@objc public class FoundEvent: PeerIDEvent {
    let message: String

    init(_ peerID: String, _ message: String) {
        self.message = message

        super.init(peerID)
    }

    override public func toJSObject() -> JSObject {
        var result = super.toJSObject()

        result["message"] = message

        return result
    }
}
